### PR TITLE
Fixes #26763 - Pass string representation of time in host info

### PR DESCRIPTION
**Pull request by tbrisker**
Otherwise Puppet 6 can't handle the host enc output (see
https://tickets.puppetlabs.com/browse/PUP-9506 for more info)

### CODE REVIEW

File: app/models/katello/host/info_provider.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def content_view_info
           'label' => host.content_view.try(:label),
           'latest-version' => host.content_view.try(:latest_version),
           'version' => content_version.try(:version),
-          'published' => content_version.try(:created_at).try(:time),
+          'published' => content_version.try(:created_at).try(:time).to_s,
```
Comment (jlsherrill) [inline app/models/katello/host/info_provider.rb]: should this be .try(:to_s)?  otherwise you'd end up with 'nil'  right?

Comment (tbrisker) [reply app/models/katello/host/info_provider.rb]:
```
[1] pry(main)> nil.to_s
=> ""

**Comment by jlsherrill** [comment]
Oh nevermind then!  I guess I was misremembering!

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
[test katello]

ci timed out

**Comment by jlsherrill** [comment]
[test katello]

transient test failure :(

**Pull request merged**